rfc11720

```diff
@@ -81,6 +124,10 @@ The objectives of the myTeams project RFC is to allow communication between myTe
     - Success
 - 01
     - Client connected
+- 02
+    - Client subscribe at a teams OK
```

Comment:
We should only have one "Success" code

Reply:
We must have only one code "Success". Maybe the type should be given in the parameters of the answer.